src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -624,7 +624,7 @@ public void setBikeBoardCost(int bikeBoardCost) {
     
     public void setPreferredAgencies(String s) {
         if (s != null && !s.equals(""))
-            preferredAgencies = new HashSet<String>(Arrays.asList(s.split(",")));
+            preferredAgencies = Sets.newHashSet(s.split(","));
```

Comment:
"This method is not actually very useful and will likely be deprecated in the future."

Reply:
Witch method is "likely be deprecated in the future" ? `Arrays.asList`?